CONTRIBUTING.md

```
@@ -0,0 +1,61 @@
+
+1. Create a feature branch from `main`.
+2. Make focused changes — one concern per commit.
+3. Run the full validation gate before pushing:
+
+```bash
+pnpm validate    # structural integrity (required)
+pnpm lint        # ESLint
+pnpm test        # Jest (2113+ tests)
+```
+
+4. Open a PR with a clear summary, test results, and screenshots for UI changes.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)